numpy/ma/tests/test_extras.py

```diff
@@ -791,13 +821,36 @@ def test_neg_axis(self):
         x[:3] = x[-3:] = masked
         assert_equal(median(x, axis=-1), median(x, axis=1))
 
+    def test_out_1d(self):
+        # integer float even odd
+        for v in (30, 30., 31, 31.):
+            x = masked_array(np.arange(v))
+            x[:3] = x[-3:] = masked
+            out = masked_array(np.ones(()))
+            r = median(x, out=out)
+            if v == 30:
+                assert_equal(out, 14.5)
+            else:
+                assert_equal(out, 15.)
+            assert_(r is out)
+            assert_(type(r) == MaskedArray)
```

Comment:
I think @charris just meant that it should be `type(r) is MaskedArray` (which I think is correct).